tensorflow/compiler/tf2tensorrt/segment/segment.cc

```diff
@@ -757,6 +767,10 @@ Status SegmentGraph(const Graph* tf_graph,
                OperationHasDynamicNonBatchDimension(graph_properties,
                                                     node->tf_node())) {
       exclude_node("dynamic non-batch dimensions not allowed");
+    } else if (UnsupportedResize(node)) {
+      //NOTE: align_corners=False is not supported by TRT as of 7.1,
```

Comment:
I think we should move this checking to ConvertResize convert_nodes.cc.

Reply:
Testing the changes with the specified change, will push them once validated.